src/Components/Panels/PanelMenu.tsx

```diff
@@ -47,10 +47,12 @@ interface PanelMenuState extends SceneObjectState {
   body?: VizPanelMenu;
   frame?: DataFrame;
   labelName?: string;
+  getLabelName?: () => string;
```

Comment:
getLabelName in a PanelMenu sounds generic, when it looks like this is specific to investigations, could we create an `investigationsOptions` or `ExplorationsButtonOptions` object, and maybe rename getLabelName to getPanelName?